lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -60,12 +65,14 @@
                             view: new AccountSettingsFieldViews.PasswordFieldView({
                                 model: userAccountModel,
                                 title: gettext('Password'),
-                                screenReaderTitle: gettext('Reset your Password'),
+                                screenReaderTitle: gettext('Reset Your Password'),
                                 valueAttribute: 'password',
                                 emailAttribute: 'email',
-                                linkTitle: gettext('Reset Password'),
+                                linkTitle: gettext('Reset Your Password'),
                                 linkHref: fieldsData.password.url,
-                                helpMessage: gettext('When you click "Reset Password", a message will be sent to your email address. Click the link in the message to reset your password.')
+                                helpMessage: gettext('When you click "Reset Your Password", edX will send a message ' +
```

Comment:
@waheedahmed I ran across this when rebasing another PR and there are a couple of issues with this logic:
 - the Transifex parser cannot understand ``gettext`` calls that are split across multiple lines so this won't localize correctly.
 - our help text should never have ``edX`` in the message because this code is not only used by edX. The message should either be generic, or else the product name should be passed through as a parameter.